examples/yaml/control_rrfs_cmaq_airnow_norm.yaml

```diff
@@ -0,0 +1,226 @@
+        #CO: 'CO'
+        #NO2: 'NO2'
+        #'NO': 'NO' #Be careful any python commands like no need to be sent in as a string.
+        PM25_TOT: 'PM2.5'
+        O3: 'OZONE'
+    projection: None
+    plot_kwargs: #Opt
+      color: 'magenta'
+      marker: 'o'
+      linestyle: '-'
+  cmaq_nacc: # model label
+    files: /scratch2/BMC/rcm1/rhs/fv3/regional/processed/v150_a/data/cmaq_nacc/aqm.201908*
+    mod_type: 'cmaq'
+    mod_kwargs:
+      concatenate_forecasts: True #Set this to true if combining forecasts together for CMAQ only.
+    radius_of_influence: 18000 #meters: horizontal resolution * 1.5
+    mapping: #model species name : obs species name
+      airnow:
+        #CO: 'CO'
+        #NO2: 'NO2'
+        #'NO': 'NO' #Be careful any python commands like no need to be sent in as a string.
+        PM25_TOT: 'PM2.5'
+        O3: 'OZONE'
+    projection: None
+    plot_kwargs: #Opt
+      color: 'cyan'
+      marker: 's'
+      linestyle: '-'
+  rrfs_v150_a: # model label
+    files: /scratch2/BMC/rcm1/rhs/fv3/regional/processed/v150_a/data/rrfs_v150_a/201908*dynf*.nc
+    files_pm25: /scratch2/BMC/rcm1/rhs/fv3/regional/processed/v150_a/data/rrfs_v150_a/201908*pm25tot*.nc
+    mod_type: 'rrfs'
+    mod_kwargs:
+      mech: 'cb6r3_ae6_aq'
+      surf_only: True
+    radius_of_influence: 19500 #meters: horizontal resolution * 1.5
+    mapping: #model species name : obs species name
+      airnow:
+        #co: 'CO'
+        #no2: 'NO2'
+        #'no': 'NO' #Be careful any python commands like no need to be sent in as a string.
+        PM25_TOT: 'PM2.5'
+        o3: 'OZONE'
+    projection: None
+    plot_kwargs: #Opt
+      color: 'goldenrod'
+      marker: '^'
+      linestyle: '-'      
+obs:
+  airnow: # obs label
+    use_airnow: True
+    filename: /scratch2/BMC/rcm1/rhs/obs/Surface/MELODIES-MONET/AIRNOW_20190801_20190831.nc
+    obs_type: pt_sfc
+    variables: #Opt 
+      PM2.5:
+        unit_scale: 1
+        unit_scale_method: '*' # Multiply = '*' , Add = '+', subtract = '-', divide = '/'
+        #obs_min: 0 # set all values less than this value to NaN
+        #obs_max: 100 # set all values greater than this value to NaN
+        nan_value: -1.0 # Set this value to NaN
+        #The obs_min, obs_max, and nan_values are set to NaN first and then the unit conversion is applied.
+        ylabel_plot: 'PM2.5 (ug/m3)' #Optional to set ylabel so can include units and/or instr etc.
+        ty_scale: 3.0 #Opt
+        vmin_plot: 0.0 #Opt Min for y-axis during plotting. To apply to a plot, change restrict_yaxis = True.
+        vmax_plot: 22.0 #Opt Max for y-axis during plotting. To apply to a plot, change restrict_yaxis = True.
+        vdiff_plot: 15.0 #Opt +/- range to use in bias plots. To apply to a plot, change restrict_yaxis = True.
+        nlevels_plot: 23 #Opt number of levels used in colorbar for contourf plot.
+        regulatory: False #Opt compute regulatory functions
+        ylabel2_plot: 'PM2.5_24hr (ug/m3)'
```

Comment:
Can we change here and in the code this to ylabel_reg_plot? Just so that it is clearer what it is for.